### DISCUSSION TITLE
When Chia Pools Start? - Updates Welcome

### DISCUSSION THREAD

**Discussion by Philipmcnill** [Chia-Network/chia-blockchain#7016] [General]
Hi,
If anyone knows when Chia pools officially start, if there are some safe pools people can join now please advice. Any updates?
Thanks

**Comment by loppefaaret** [comment]
we are close to entering phase 2 of the official pooling protocol - which means getting the reference pool design finalized, and opened up on for public testing, and having various pool operators test out their implementations, and gather data - but when it comes to any dates, we are not going to announce any firm dates! in that regards, it is ready when it is ready - soon

**Reply by Philipmcnill** [reply]
Thanks for answer. Just do not want to missed it. I understand when it is open for public testing there will be some update on official Chia network website?

**Reply by loppefaaret** [reply]
If you want to keep a close eye on it, the real place is our Keybase chat:
https://keybase.io/team/chia_network.public
on there we apart from the #general chat, also have a #announcements and a #pools chat
ofcourse there is also watching the branches here on our github

**Comment by alexwwstx** [comment]
hpool seems the best one if you really want to join

**Reply by Philipmcnill** [reply]
I think they have their own protocol and also there are not predictable things happening in China with crypto miners maybe also (farmers soon). I prefer to wait for official.

**Reply by joelrb** [reply]
> hpool seems the best one if you really want to join

Don't use hpool. Better join Foxy-Pool for now. They are more trustworthy and their code is open source. They use existing solo plots and they don't get your keys. This is not another Chia project/fork they just enables pooling for current plots while waiting for the official pool protocol.

https://chia-og.foxypool.io/info

**Reply by Philipmcnill** [reply]
How is that possible I can use my existing plots?

**Comment by lhcdims** [comment]
I heard from a Chinese chat group that the "migpool" is operating a chia pool using the new plotting protocol, is that true?

**Reply by Philipmcnill** [reply]
Thanks, I will have a look. However base on what loppefaaret says it would be better to wait for official project.

**Reply by loppefaaret** [reply]
there are several pools out on mainnet, that took the developer testnet release of the proposed final reference design, and started pools that runs on mainnet... they are "legit" in the sense that they are following the reference design, with partial proofs preventing faking space, and are letting the farmers build the blocks, and are using plots against nft singletons for farming in the pool... 
_**atleast, that is the idea they are giving**_ i am **NOT** green lighting these pools, and **I** will never green light _any_ pool
the responsibility solely is between the farmer and the pool in the end!
furthermore, the official stance currently on the developer release of the testnet pooling protocol, is there is up to a 25% chance, that any plots made by this release, will end up useless, if we find a potential breaking bug, that causes us to change the fundamentals of the design!
so joining and plotting for these early bird pools, is a bit of gambling... there is some that likes living on the edge, but be aware of the above caveats!

**Reply by Philipmcnill** [reply]
Thanks for that. Great update

**Comment by Philipmcnill** [comment]
I see the release new Chia 2.0 - is there any pool I can move with my current plots?

**Reply by loppefaaret** [reply]
our newest release is 1.2.0 - the Official Pooling release
it requires new plots created to join pools that uses the newly released official pooling protocol
although i can not recommend it, there are some pools that use their own code, enabling using the old OG plots to farm.
it would require to download and use 3rd party code - and is strongly discouraged from Chia Network

**Reply by Philipmcnill** [reply]
So what is the best to do now. Do you recommend to stay a keep going with a standard original way?

**Reply by loppefaaret** [reply]
no, Chia Network has made a conscious decision _not_ to recommend any pools - and we absolutely do not endorse pooling with OG plots!
the best i can do for you, is tell you that you can find a list of pools here:
https://miningpoolstats.stream/chia

**Reply by Philipmcnill** [reply]
It makes sense. Thanks
